packages/appstore/src/modules/onboarding/__test__/empty-onboarding.spec.tsx

```diff
@@ -11,6 +11,20 @@ jest.mock('Assets/svgs/trading-platform', () => ({
     default: () => <div>DerivLogo</div>,
 }));
 
+
+jest.mock('@deriv-com/analytics', () => ({
+    Analytics: {
+        getFeatureValue: jest.fn(),
+        getInstances: jest.fn().mockReturnValue({
+            ab: {
+                GrowthBook: {
+                    setRenderer: jest.fn(),
+                },
+            },
+        }),
+    },
+}));
```

Comment:
Analytical package mock exist in `globals.js`
Could you please try to move these features inside globals file?
I'm not sure, but it should works for us

Reply:
Good idea, let me do it